pkgs/development/python-modules/fastmcp/default.nix

```diff
@@ -49,10 +53,12 @@ buildPythonPackage rec {
 
   dependencies = [
     authlib
+    cyclopts
     exceptiongroup
     httpx
     mcp
     openapi-pydantic
+    pyperclip
     python-dotenv
     rich
     typer
```

Comment:
No longer depends on `typer`, also new dependency on `pydantic` and `pydantic.optional-dependencies.email`